packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageTableColumnHeader.scss

```
@@ -0,0 +1,6 @@
+.wallets-dynamic-leverage-screen {
+    &__market-table-header-cell {
```

```suggestion
    &__table-cell {
```
This looks too long for a classname 😱

---

Updated.